plugin/src/test/groovy/com/novoda/staticanalysis/internal/checkstyle/CheckstyleAndroidVariantIntegrationTest.groovy

```diff
@@ -11,30 +12,15 @@ import static com.novoda.test.LogsSubject.assertThat
 
 class CheckstyleAndroidVariantIntegrationTest {
 
-    private static final String DEFAULT_CHECKSTYLE_CONFIG = "checkstyle { configFile new File('${Fixtures.Checkstyle.MODULES.path}') }"
+    private static final String DEFAULT_CHECKSTYLE_CONFIG = """
+        checkstyle { configFile new File('${Fixtures.Checkstyle.MODULES.path}') }
+    """
```

Comment:
because of the string interpolation (ie: `${...}`), you need to use `GString` (quoted in `""`) instead of simple `String` (using `''`)